tester/tester_loop.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+let i=0;
+let nb_failed=0;
+for TEST in $tests_path/*; do
+	((i+=1))
+	expected_out=$(eval bash $TEST 2>&1)
+	test_out=$(eval ${prog_pathname} $TEST 2>&1)
+
+	TEST_COLOR="$GREEN"
+	OK_OUT=1;
+
+	if [ "$expected_out" != "$test_out" ]; then
+		TEST_COLOR="$RED"
+		OK_OUT=0
+	fi;
+
+	echo -e "\n 🌴🥥------------------------------🥥🌴\n"
```

Comment:
love the cocoshells emojis, we should use them in our Makefile as well

Reply:
I agree very much.